### PR TITLE
feat: logout flow

### PR DESCRIPTION
**Pull request by rebelchris**
Add a logout flow state for legacy users based on version

Wait with merging until we know the version

### CODE REVIEW

File: src/routes/boot.js [hunk 1]
```diff
@@ -155,68 +157,86 @@ export const bootSharedLogic = async (ctx, shouldRefreshToken) => {
...
-            providers: [userProvider?.provider],
-            roles,
-            permalink: `${config.webappOrigin}/${user.username || user.id}`,
-          },
-          accessToken,
-        };
-        if (!user.infoConfirmed) {
+
+    const { v: appVersion } = ctx.request.query;
+    if (appVersion && appVersion.localeCompare(process.env.USER_LOGOUT_VERSION, undefined, { numeric: true, sensitivity: 'base' }) >= 0 && !isKratos) {
```
Comment (idoshamun) [inline src/routes/boot.js:162]: Can we turn it into a function for better readability, please?

**Review by idoshamun — APPROVED** [review]
Overall looks good just one comment

File: src/routes/boot.js (hunk 1 above)
Comment (rebelchris) [reply src/routes/boot.js:162]: Turned this into a function

**Comment by idoshamun** [comment]
We need to add a pulumi variable to point to the right version before merging this one

**Review by idoshamun — APPROVED** [review]

**Pull request merged**